src/rust/uk.mangainua/src/helper.rs

```diff
@@ -0,0 +1,75 @@
+use aidoku::std::String;
+
+pub fn is_nsfw(genre: String) -> bool {
+	let nsfwcategories = "Еччі Юрі Яой 18+"; // maybe new
+	if nsfwcategories.contains(&genre) {
+		return true;
+	}
+	//return false;
+	false
+}
```

Comment:
just pass by reference 
```suggestion
pub fn is_nsfw(genre: &String) -> bool {
	let nsfwcategories = "Еччі Юрі Яой 18+"; // maybe new
	nsfwcategories.contains(genre)
}
```